webapp/channels/src/components/admin_console/custom_enable_disable_guest_accounts_setting.tsx

```diff
@@ -3,9 +3,9 @@
 
 import React, {useCallback} from 'react';
 import {FormattedMessage} from 'react-intl';
+import {Link} from 'react-router-dom';
```

Comment:
0/5 For simpler ability to change the library in the future if ever, and have a better parallelism between "ExternalLink" and "InternalLink", I wonder if we should create a wrapper for this as "InternalLink".

Reply:
Agreed 👍🏽